1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java

```diff
@@ -95,6 +99,17 @@ private void handleClientCommandCompletion(String full, final String ignored, Ca
         }
     }
 
+    /**
+     * Add this callback, to check if the User complete a Playername or a Liquidbounce command.
+     * Bug report & more info, see: https://github.com/CCBlueX/LiquidBounce1.8-Issues/issues/3795
```

Comment:
Commit messages don't go in comments

Reply:
Okay, should i remove the whole comment or only the commit url?

Reply:
I think this comment is fine.